newReview.php

```diff
@@ -0,0 +1,39 @@
+<?php
+session_start();
+$game = $_GET['game'];
+$user = $_GET['user'];
+
+require_once  'navbar.php';
+
+require_once 'dbInit.php';
```

Comment:
The convention is to put the requires after the session_start, but before anything else, so move these up